doc/build-osx.md

```diff
@@ -54,6 +54,12 @@ brew install automake libtool boost pkg-config libevent
 ### 4. Clone Bitcoin repository
 
 `git` should already be installed by default on your system.
+If it is not, install it with the following command:
+
+``` bash
+brew install git
```

Comment:
I haven't tried reinstalling macOS recently, but this seems unlikely, because homebrew itself uses Git. It does ship a more recent version, but that's only useful for codereview.

Reply:
Agree. On my system, the `/usr/bin/git` is not a Homebrew's package.

Reply:
Indeed on M1 macs, brew has a whole new root directory:

```
--> which git
/opt/homebrew/bin/git
```